File: sed_repos/java/SED7/src/main/java/ic/doc/WeatherDay.java
package ic.doc;

import com.weather.Day;

public enum WeatherDay {

  MONDAY(Day.MONDAY),
  TUESDAY(Day.TUESDAY),
  WEDNESDAY(Day.WEDNESDAY),
  THURSDAY(Day.THURSDAY),
  FRIDAY(Day.FRIDAY),
  SATURDAY(Day.SATURDAY),
  SUNDAY(Day.SUNDAY);

  private final com.weather.Day day;

  WeatherDay(com.weather.Day day) {
    this.day = day;
  }

  public Day getDay() {
    return day;
  }
}

File: sed_repos/python/SED7/src/weather_day.py
from enum import Enum
from lib.weather import Day


class WeatherDay(Enum):
    MONDAY = Day.MONDAY
    TUESDAY = Day.TUESDAY
    WEDNESDAY = Day.WEDNESDAY
    THURSDAY = Day.THURSDAY
    FRIDAY = Day.FRIDAY
    SATURDAY = Day.SATURDAY
    SUNDAY = Day.SUNDAY

    def get_day(self):
        return self.value
